src/python_tools/requirements.txt

```diff
@@ -8,3 +8,10 @@ ipywidgets==7.6.2
 jupytext==1.11.2
 tftpy==0.8.1
 pandas==1.3.4
+Adafruit-Blinka==6.15.0
+adafruit-circuitpython-busdevice==5.1.1
+adafruit-circuitpython-rgb-display==3.10.9
+Adafruit-PlatformDetect==3.18.0
+Adafruit-PureIO==1.1.9
+Jetson.GPIO==2.0.17
+Pillow==8.4.0
```

Comment:
ditto comment above, move this to `software/jetson_nano/display/requirments.txt`

Reply:
I created a `requirements.txt` file in the `software/jetson_nano/display/` folder. CI is failing because with the following error message: 
(22:13:18) ERROR: /home/runner/work/Software/Software/src/software/jetson_nano/display/BUILD:5:10: no such package '@python_tools_deps//pypi__jetson_gpio': BUILD file not found in directory 'pypi__jetson_gpio' of external repository @python_tools_deps. Add a BUILD file to a directory to mark it as a package. and referenced by '//software/jetson_nano/display:rotary_encoder'

Reply:
Fixed